ansible/roles/setup_todo_app/handlers/main.yaml

```diff
@@ -0,0 +1,6 @@
+- name: enable_todo_service
+  systemd:
+    name: todoapp
+    state: started
```

Comment:
You would want state: restarted if there are app changes.

Reply:
Good spot